src/coreclr/jit/codegenlinear.cpp

```diff
@@ -346,6 +346,9 @@ void CodeGen::genCodeForBBlist()
         if ((block->bbPrev != nullptr) && (block->bbPrev->bbJumpKind == BBJ_COND) &&
             (block->bbWeight != block->bbPrev->bbWeight))
         {
+            JITDUMP("Adding label due to BB weight difference: BBJ_COND " FMT_BB " with weight " FMT_WT
```

Comment:
While you are here, I am wondering if there is an easy way to add towards the end the mapping of `BB -> IG` somehow? While looking at JITDump, it is not straightforward to find this mapping and is very helpful overall when we look backward in JITDump.